rest_framework/authtoken/models.py

```diff
@@ -1,26 +1,22 @@
 import binascii
 import os
 
-from django.conf import settings
+from django.conf import AUTH_USER_MODEL
```

Comment:
Could you elaborate why importing `AUTH_USER_MODEL` directly ?
Django documentation imports settings and use settings.AUTH_USER_MODEL.